CHANGELOG.md

```diff
@@ -6,6 +6,11 @@
 
 * **Metrics**
   * [DEPRECATION] For security reasons, access to Prometheus and AlertManager via NodePort is no longer enabled by default. Set the environment variable PROM_NODEPORT_ENABLE=true to replicate previous behavior.
+  * [UPGRADE] - Kube-prometheus-stack has been upgraded from version 36.6.1 to 41.7.3
+  * [UPGRADE] - Prometheus has been upgraded from version 2.36.2 to 2.39.0
+  * [UPGRADE] - Prometheus Operator has been upgraded from version 0.57.0 to 0.60.0
+  * [UPGRADE] - Grafana has been upgraded from version 9.0.3 to 9.2.3
```

Comment:
Are the versions of the thing we are updating the "default" versions deployed by this new version of the kube-prom-stack Helm chart?  

Are we upgrading Alertmanager, Kube State Metrics and Pushgateway this time?  While we are primarily focused on the one CVE, I think we want to be consistent in how we upgrade things as much as possible.  So, we probably should bump these other components to whatever their default versions are based on the new version of the kube-prom-stack Helm chart.

Reply:
I bumped the version of components that happened to have updates.  Kube State Metrics did have an update (and forgot to note it as you saw); however, Alertmanager and the Pushgateway did not have any available updates since the last time I performed the update.
It will be added in the next push.